OJP/OJP_Trips.xsd

```diff
@@ -471,12 +476,17 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:group ref="OperatingDaysGroup" minOccurs="0"/>
+			<xs:group ref="TripStatusGroup" minOccurs="0"/>
```

Comment:
This merge moved the position of TripStatusGroup. Was this on purpose?

Reply:
Please report all collateral damage due to manual merging to @ue71603

Reply:
@sgrossberndt and the fact that this change was not picked up by CI shows we lack examples.

Reply:
I did it on purpose. In all other examples Situations were the last before Extension. It thought it was a mistake during the change in Situations.  Certainly I should not have done it in this PR if that was not the case (as it seems), but from homogenity, I think it is right.  I can undo it in an additional PR if you like @sgrossberndt

Reply:
No, don't undo it. But it should not have been part of this PR but handled separately.

Reply:
I know... I only did it, because I thought it was messed up during the merge by me or during the creation of the PR. Should have checked.